examples/db-templates/mongodb-ephemeral-template.json

```diff
@@ -49,7 +49,11 @@
       "spec": {
         "strategy": {
           "type": "Recreate",
-          "resources": {}
+            "resources": {
+		"limits": {
+		    "memory": "${MEMORY_LIMIT}"
```

Comment:
I see you've set this on all the deployments.  this is the amount of memory the deployment pod is going to get to perform the deployment(at least i think that's what it is).  I don't know that
1) it should be the same as the amount the application pod gets
2) it needs to be set at all

but i'm guessing you guys have had discussions about that?  similar to the build limits, it's a limit for a runonce pod as opposed to a long running pod.